public/js/show.js

```diff
@@ -0,0 +1,82 @@
+$(document).ready(function() {
+    $("#timeslot-list").hide();
+    $("#timeslot-latest").show();
+
+    var filterSeason = function(object) {
+        $(".timeslot-filter-season").removeClass("active");
+        $(object).addClass("active");
+        $("#timeslot-latest").hide();
+        $("#timeslot-season-title").text("Season " + $(object).find("td:nth-of-type(2)").html());
+        timeslotTable
+            .columns( 0 )
+            .search( $(object).data("seasonid") )
+            .draw();
+
+        $("#timeslot-list").show();
+    };
+
+    //Must run before dataTables to apply to all season/timeslot pages.
+    $(".timeslot-filter-season").on( "click", function () {
+        filterSeason(this);
+    });
+    $("#timeslot-list tbody tr").on("click", function () {
+        window.location.href = $(this).find("a").attr("href");
+    });
+
+    var seasonTable = $("#seasons").DataTable({
+        "info": false,
+        "lengthChange": false,
+        "pageLength": 5,
+        "order": [[1, "asc"]],
+        "sDom": "<\"top\"i>rt<\"bottom\"lp><\"clear\">",
+        "columnDefs": [
+        {
+            "targets": [ 2,3 ],
+            "orderable": false
+        }
+        ]
+    });
+    var timeslotTable = $("#timeslots").DataTable({
+        "info": false,
+        "lengthChange": false,
+        "sDom": '<"top"i>rt<"bottom"lp><"clear">',
+        "pageLength": 5,
```

Comment:
5 seems a little short, maybe 10? Most seasons are going to be 10 episodes anyway

Reply:
Fair, it was such that all the columns were about the same height, but yeah.